bftclient/include/bftclient/metrics.h

```diff
@@ -0,0 +1,43 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "Metrics.hpp"
+#include "base_types.h"
+
+namespace bft::client {
+
+struct Metrics {
```

Comment:
Just curious - why `struct`, given that you still define member variable visibility?

Reply:
No great reason. It's what I initially had before adding various visibility specifiers and reordering things. One decent reason though is intention: I personally view structs as meaning we want to access fields directly without getters/setters. Whether this distinction actually makes sense is questionable. However, the Google C++ style guide seems to agree since trailing underscores aren't used in struct members, which is what I wanted.

Reply:
Thanks for the clarification!